hackathon_site/registration/models.py

```diff
@@ -25,23 +28,94 @@ def __str__(self):
 
 
 class Application(models.Model):
-    user = models.ForeignKey(User, on_delete=models.CASCADE, null=False)
-    team = models.ForeignKey(Team, on_delete=models.CASCADE, null=False)
+    GENDER_CHOICES = [
+        (None, ""),
+        ("male", "Male"),
+        ("female", "Female"),
+        ("non-binary", "Non-binary"),
+        ("other", "Other"),
+        ("no-answer", "Prefer not to answer"),
+    ]
+
+    ETHNICITY_CHOICES = [
+        (None, ""),
+        ("american-native", "American Indian or Alaskan Native"),
+        ("asian-pacific-islander", "Asian / Pacific Islander"),
+        ("black-african-american", "Black or African American"),
+        ("hispanic", "Hispanic"),
+        ("caucasian", "White / Caucasian"),
+        ("other", "Multiple ethnicity / Other"),
+        ("no-answer", "Prefer not to answer"),
+    ]
+
+    STUDY_LEVEL_CHOICES = [
+        (None, ""),
+        ("highschool", "High School"),
+        ("undergraduate", "Undergraduate"),
+        ("gradschool", "Graduate School"),
+        ("other", "Other"),
+    ]
+
+    user = models.OneToOneField(User, on_delete=models.CASCADE, null=False)
+    team = models.ForeignKey(
```

Comment:
Shouldn't this team be one to one is what I meant

Reply:
No, see my reply above